Stylesheets/components/button.css

```diff
@@ -0,0 +1,294 @@
+@import url("/stylesheets/main.css");
+
+.btn-hover {
+  background-color: black;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  min-width: 100vh;
+  height: 100px;
+}
+
+.btn-1 {
+  width: 200px;
+  height: 60px;
```

Comment:
instead of giving a fixed width and height to the button, try using padding. Because if you give it fixed dimensions, then the content can sometimes overflow, but with padding, it would scale according to the content and never overflow

Reply:
oh yes ty. I made changes as you suggested